addons/web/static/src/views/kanban/kanban_record.js

```diff
@@ -202,6 +202,12 @@ export class KanbanRecord extends Component {
         onWillUpdateProps(this.createRecordAndWidget);
     }
 
+    getFormattedValue(fieldName, fieldId) {
```

Comment:
you don't need the fieldName anymore as you can obtain it from `archInfo.fieldNodes[fieldId].name`